server/test/models/address_test.rb

```diff
@@ -34,13 +34,66 @@ class AddressTest < ActiveSupport::TestCase
     end
   end
 
-  test ".find_or_create_address should return the address when the address_hash exists" do
+  test "address_hash should be nil when args is invalid" do
+    VCR.use_cassette("blocks/10") do
```

Comment:
I want to know whether setting "blocks/10" as constant to reuse is a better action in ruby ?

Reply:
blocks/10 is point to a real file named `10.yml`, in test I think can be left as it is.

Reply:
OK